### PR TITLE
update docstrings

### PR DESCRIPTION
**Pull request by AshtonSBradley**
- consistent order of External links and See also
- minor clarifications for ellipk/ellipe regarding domain of argument.

### CODE REVIEW

File: src/ellip.jl [hunk 1]
```diff
@@ -15,14 +15,13 @@ Computes Complete Elliptic Integral of 1st kind ``K(m)`` for parameter ``m`` giv
 \quad \text{for} \quad m \in \left( -\infty, 1 \right] \, .
 ```
 
-External links: [DLMF](https://dlmf.nist.gov/19.2#E8),
-[Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants).
+External links: [DLMF](https://dlmf.nist.gov/19.2#E8), [Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants).
```
Comment (simonbyrne) [inline src/ellip.jl]: You can link to DLMF via X.Y.Z format:

```suggestion
External links: [DLMF 19.2.8](https://dlmf.nist.gov/19.2.8), [Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants).
```

File: src/ellip.jl [hunk 2]
```diff
@@ -191,14 +190,13 @@ Computes Complete Elliptic Integral of 2nd kind ``E(m)`` for parameter ``m`` giv
 \quad \text{for} \quad m \in \left( -\infty, 1 \right] \, .
 ```
 
-External links: [DLMF](https://dlmf.nist.gov/19.2#E8),
-[Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Complete_elliptic_integral_of_the_second_kind).
+External links: [DLMF](https://dlmf.nist.gov/19.2#E8), [Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Complete_elliptic_integral_of_the_second_kind).
```
Comment (simonbyrne) [inline src/ellip.jl]:
```suggestion
External links: [DLMF 19.2.8](https://dlmf.nist.gov/19.2.8), [Wikipedia](https://en.wikipedia.org/wiki/Elliptic_integral#Complete_elliptic_integral_of_the_second_kind).
```

**Comment by AshtonSBradley** [comment]
i've updated all DLMF links to X.Y.Z form

**Comment by simonbyrne** [comment]
Thanks!

**Pull request merged**